crates/core/tedge_agent/src/entity_manager/tests.rs

```diff
@@ -196,19 +196,23 @@ mod model {
 
             for command in &self.0 {
                 f.write_str(sep)?;
-                f.write_str(format!("{command}").as_str())?;
                 if f.alternate() {
-                    sep = "\n    "; // On test unit output, print each command on a new line
+                    // On test unit output, print each command on a new line (using Rust notation)
+                    sep = "\n    ";
+                    f.write_str(format!("// {command}{sep}").as_str())?;
```

Comment:
I was wondering why this ron representation wasn't added at the `Command` level itself and not limited to `Commands`, so that we get these visual improvements even when we have any other collection of `Command` instances . I'm assuming it was done intentionally to keep the `tests.txt` content minimal with just one representation.

Reply:
> I'm assuming it was done intentionally to keep the tests.txt content minimal with just one representation.

Yes. I try to keep `test.txt` as simple as possible